src/mudlet.cpp

```diff
@@ -2599,7 +2599,6 @@ void mudlet::deleteProfileData(const QString& profile, const QString& item)
     }
 }
 
-// this slot is called via a timer in the constructor of mudlet::mudlet()
```

Comment:
Humm, it seems that this wasn't the case but now it is - so is the comment not needed? (The comment itself dates to a commit in January 2009 by Heiko: 36e620ed595c3dcc5bdf530204b7bc25156efe4a!)